docs/docs-content/vertex/system-management/enable-non-fips-settings/enable-non-fips-settings.md

```diff
@@ -12,12 +12,12 @@ tags: ["vertex", "non-fips"]
 
 Palette VerteX is FIPS-enforced by default, incorporating the Spectro Cloud Cryptographic Module into the Kubernetes Management Platform and the infrastructure components of target clusters. To learn more about our cryptographic library, check out [FIPS 140-2 Certification](../../../compliance.md#fips-140-2).
 
-If desired, you can allow the consumption of certain non-FIPS functionality in Palette VerteX at the tenant level. **Platform Settings** at the tenant level provide toggles to allow non-FIPS-compliant add-on packs and non-FIPS features such as scans, backup, and restore. You can also allow importing clusters created external to Palette.
+If desired, you can allow the consumption of certain non-FIPS functionality in Palette VerteX at the tenant level. **Platform Settings** at the tenant level provide toggles to allow non-FIPS-compliant packs and non-FIPS features such as scans, backup, and restore. You can also allow importing clusters created external to Palette.
 
 
 ## Resources
 
-- [Use non-FIPS Add-On Packs](../../system-management/enable-non-fips-settings/use-non-fips-addon-packs.md)
+- [Use non-FIPS Packs](../../system-management/enable-non-fips-settings/use-non-fips-addon-packs.md)
```

Comment:
The URL will still mention addon packs, can it be changed?

Reply:
@cloudmaniac we could change the URL, but it's one of those things that is not necessary. We try to avoid redirects when possible.